modules/sbo-binding-workloads-that-are-not-compliant-with-PodSpec.adoc

```diff
@@ -0,0 +1,151 @@
+// Module included in the following assemblies:
+//
+// * /applications/connecting_applications_to_services/binding-workloads-using-sbo.adoc
+
+[id="sbo-binding-workloads-that-are-not-compliant-with-PodSpec_{context}"]
+= Binding secondary workloads that are not compliant with PodSpec
+
+A typical scenario in service binding involves backing service, workload (Deployment), and {servicebinding-title}. Consider a scenario that involves a secondary workload, which can also be an application Operator, not compliant with PodSpec and is between the primary workload (Deployment) and {servicebinding-title}.
+
+For such secondary workload resources, the location of the container path is arbitrary. For service binding, if the secondary workload in a CR is not compliant with the PodSpec, you must specify the location of the container path. This helps in projecting the binding data into the container path specified in the secondary workload of the `ServiceBinding` CR, for example, when you do not want the binding data inside a pod.
```

Comment:
```suggestion
For such secondary workload resources, the location of the container path is arbitrary. For service binding, if the secondary workload in a CR is not compliant with the PodSpec, you must specify the location of the container path. Doing so projects the binding data into the container path specified in the secondary workload of the `ServiceBinding` CR, which you might want to do if you do not want the binding data inside a pod.
```
- `This helps` - What does `This` refer to in this sentence? There's no direct antecedent, so I'm not sure what the sentence is referring to. See the "Grammar" section of the ISG for more information: e.g. `Make the subject of a sentence clear.`
- `helps` doesn't seem like the right word to use if the specifying the location of the container path is required.
- My edit shows one possible way to clarify the relationship between the different pieces of information, though maybe it alters the meaning too much.

Reply:
Yes, agree, thanks for pointing this out. I have implemented "Doing so projects". Retaining the last sentence as is as changing this to  "Which you can do if" alters the meaning.